### PR TITLE
library: add cephadm_bootstrap module

### PR DESCRIPTION
**Pull request by dsavineau**
This adds cephadm_bootstrap ansible module for replacing the command module
usage with the cephadm bootstrap command.

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>

### CODE REVIEW

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

File: infrastructure-playbooks/cephadm.yml [hunk 1]
```diff
@@ -143,10 +143,16 @@
...
-      command: "{{ cephadm_cmd }} bootstrap --mon-ip {{ _current_monitor_address }} --skip-pull --skip-monitoring-stack {{ '--initial-dashboard-user ' + dashboard_admin_user + ' --initial-dashboard-password ' + dashboard_admin_password if dashboard_enabled | bool else '--skip-dashboard' }}"
-      changed_when: false
-      environment:
-        CEPHADM_IMAGE: '{{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'
+      cephadm_bootstrap:
+        mon_ip: "{{ _current_monitor_address }}"
+        image: "{{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}"
+        docker: "{{ true if container_binary == 'docker' else false }}"
+        pull: false
+        dashboard: "{{ true if dashboard_enabled | bool else false }}"
```
Comment (guits) [inline infrastructure-playbooks/cephadm.yml:151]: why do we need this given there's a default in ceph-defaults for `dashboard_enabled` ?

```suggestion
        dashboard: "{{ dashboard_enabled }}"
```

File: infrastructure-playbooks/cephadm.yml [hunk 2]
```diff
@@ -143,10 +143,16 @@
...
+        dashboard_user: "{{ dashboard_admin_user if dashboard_enabled | bool else omit }}"
+        dashboard_password: "{{ dashboard_admin_password if dashboard_enabled | bool else omit }}"
+        monitoring: false
+        firewalld: "{{ true if configure_firewall | bool else false }}"
```
Comment (guits) [inline infrastructure-playbooks/cephadm.yml:155]: ditto

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Comment by dsavineau** [comment]
jenkins test centos-container-cephadm

**Review by guits — APPROVED** [review]

**Pull request merged**